client/src/pages/GalleryPage.js

```diff
@@ -10,31 +10,27 @@ const Display = styled.div`
 
 const GalleryPage = () => {
   const userName = "sven";
-  //const tag = "2";
-  const { data, loading, error, doFetch } = useAsync(() =>
+  const { data: userData, loading, error, doFetch } = useAsync(() =>
     getImageObj(userName)
   );
 
   useEffect(() => {
     doFetch();
```

Comment:
Not exactly sure why but your deployment fails and it seems to be a problem here ...

---
Error Message:
```
Failed to compile.

src/pages/GalleryPage.js
  Line 19:6:  React Hook useEffect has a missing dependency: 'doFetch'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
```